lib/ansible/template/__init__.py

```diff
@@ -810,16 +810,16 @@ def _now_datetime(self, utc=False, fmt=None):
 
         return now
 
-    def _query_lookup(self, name, *args, **kwargs):
+    def _query_lookup(self, name_, *args, **kwargs):
```

Comment:
We should name this to something other than `name_` such as `lookup_name` or something more verbose.

Reply:
Would it be better to get rid of named params entirely and use `args[0]` to get the name of the lookup plugin?

This way we don't break any existing lookup plugins that use `lookup_name` or whatever is chosen.

Reply:
`ansible_lookup_name` since we reserve the `ansible_` naming prefix, that way any collisions would be really hard to justify